From b18fc6fb3909736dc34e5d036a639ad9685e52e4 Mon Sep 17 00:00:00 2001
From: Gatien Chevallier <gatien.chevallier@foss.st.com>
Date: Fri, 16 Feb 2024 15:57:13 +0100
Subject: [PATCH 0604/1695] dt-bindings: shared bindings between STM32MP25x and
 STM32MP23x boards

As the STM32MP23 SoC family is a derivative of the STM32MP25 SoC family,
the device tree bindings can be shared as they are compatible.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git a4170ba38eed8dbd7456739be8bfdbc9bd7fe5e9 dt-bindings: shared bindings between STM32MP25x and STM32MP23x boards]
Signed-off-by: Gatien Chevallier <gatien.chevallier@foss.st.com>
Change-Id: Ia59c568e6357151c3278182cc227796548f6f7cc
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/360736
Reviewed-by: Gatien CHEVALLIER <gatien.chevallier@st.com>
Tested-by: Gatien CHEVALLIER <gatien.chevallier@st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Etienne CARRIERE <etienne.carriere@foss.st.com>
Domain-Review: Yann GAUTIER <yann.gautier@foss.st.com>
---
 core/include/drivers/stm32mp_dt_bindings.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/core/include/drivers/stm32mp_dt_bindings.h b/core/include/drivers/stm32mp_dt_bindings.h
index 006399e33..ae65a60a6 100644
--- a/core/include/drivers/stm32mp_dt_bindings.h
+++ b/core/include/drivers/stm32mp_dt_bindings.h
@@ -25,7 +25,7 @@
 #include <dt-bindings/soc/stm32mp15-tzc400.h>
 #endif
 
-#ifdef CFG_STM32MP25
+#if defined(CFG_STM32MP25) || defined(CFG_STM32MP23)
 #include <dt-bindings/clock/stm32mp25-clks.h>
 #include <dt-bindings/clock/stm32mp25-clksrc.h>
 #include <dt-bindings/reset/stm32mp25-resets.h>
-- 
2.47.2

